### PR TITLE
Feat: Upgrade parachain 1.21.0

### PR DESCRIPTION
**Pull request by bvotteler**
- Resolves interlay/interbtc-squid#60 : Upgrade squid to work with parachain runtime version `1021000`
- Resolves interlay/interbtc-squid#58 : Update README to show correct order to generate a SQL schema upgrade.
- Generated SQL upgrades
- Bump squid version to `0.12.0` (was `0.11.1`)

### CODE REVIEW

**Comment by bvotteler** [comment]
> #### ⚠️ GitGuardian has uncovered 1 secret following the scan of your pull request.

This is a false positive. @ns212 can I override this, or do you have to do that?

**Review by peterslany — APPROVED** [review]
Great job updating the types properly and making the code cleaner! Only comment I left is regarding the naming, which is only a suggestion that can be ignored :)

File: schema.graphql [hunk 1]
```diff
@@ -35,7 +35,18 @@ type LendToken {
...
+type StableLpToken {
+    poolId: Int!
+}
+
+union LpToken = NativeToken | ForeignAsset | StableLpToken
+
+type LpTokenPair {
+    token0: LpToken!
+    token1: LpToken!
+}
```
[the remark this answers is not in the document]
Comment (bvotteler) [reply schema.graphql:47]: > What do you think about renaming `LpToken` to `PooledToken` and then for `LpTokenPair` we could use `LpToken` consistently with the runtime?

I like that idea! Thanks for the input

**Comment by bvotteler** [comment]
Full indexing tested:
- [x] Interlay mainnet (runtime `1020001`) ✅ 
- [x] Kintsugi mainnet (runtime `1020001`) ✅ 
- [] Interlay testnet (no plans to test) 
- [x] Kintsugi testnet (runtime `1020000`) ✅

**Pull request merged**